### PR TITLE
new groovy big ints tests

### PR DESCRIPTION
**Pull request by krnone**

### CODE REVIEW

File: test/tests/beakerx.po.js [hunk 1]
```diff
@@ -135,5 +135,18 @@ var BeakerXPageObject = function () {
     expect(resultTest).toMatch(expectedText);
   }
 
+  this.getTableColumnLabel = function(tableIndex, columnIndex){
+  var table = $$("div.dataTables_scrollHead") [tableIndex]
```
Comment (scottdraves) [inline test/tests/beakerx.po.js]: code needs indents and semicolon.

File: test/tests/groovy/bigIntsTest.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+     cellText = tableColumnLabel.getText()
+     expect(cellText).toMatch('temp')
+     });
+
+    it('Table has proper cell values', function () {
+     var tableCell
+     var cellText
+     tableCell = beakerxPO.getTableCell(0, 0, 1)
+     cellText = tableCell.getText()
+     expect(cellText).toMatch('1912276048007')
```
Comment (scottdraves) [inline test/tests/groovy/bigIntsTest.js]: what are these values? when i run the notebook the values i get are:
```
1516039311030000007	1516039311030000077	14.600
1516039311030000014	3032078622060000176	18.100
1516039311030000021	4548117933090000297	23.600
```

Comment (krnone) [reply test/tests/groovy/bigIntsTest.js]: Try to run notebook from tests/notebooks/groovy folder. Notebook there is a bit different from the original one and 'milis' is changed from date function to a constant, as Vladimir suggested.

File: test/tests/beakerx.po.js (hunk 1 above)
Comment (krnone) [reply test/tests/beakerx.po.js]: Done.

**Comment by scottdraves** [comment]
using a constant for millis is a good idea, but definitely should not overflow into negative values, that's a bug. i think just making millis a "L" or "g" constant (respectively in each test) will fix it.

**Comment by scottdraves** [comment]
```

250 passing (279.80s)
3 failing

1) 64-bit Longs labels and valuessuite2 Table has proper cell values:
Expected '1912276048007' to match '123456789000007'.
running chrome
Error: Expected '1912276048007' to match '123456789000007'.
    at UserContext.<anonymous> (/Users/spot/src/beakerx/test/tests/groovy/bigIntsTest.js:57:24)

2) BigNums (arbitrary precision) labels and valuessuite3 Table has proper cell values:
Expected '-915537047993' to match '1987654321000007'.
running chrome
Error: Expected '-915537047993' to match '1987654321000007'.
    at UserContext.<anonymous> (/Users/spot/src/beakerx/test/tests/groovy/bigIntsTest.js:119:24)

3) EasyForm List fieldsuite7 Should select List value by code:
Failed: invalid selector: An invalid or illegal selector was specified
running chrome
Error: Argument was an invalid selector (e.g. XPath/CSS).
    at element("option=java.lang.IllegalArgumentException: "onef6" is not a valid option for SelectMultipleWidget "field name6".
    at com.twosigma.beakerx.easyform.EasyForm.put(EasyForm.java:315)
    at com.twosigma.beakerx.easyform.EasyForm.put(EasyForm.java:46)
    at script15167983090621327029612.run(script15167983090621327029612.groovy:1)
    at com.twosigma.beakerx.groovy.evaluator.GroovyCodeRunner.run(GroovyCodeRunner.java:71)") - $.js:42:17
    at $("option=java.lang.IllegalArgumentException: "onef6" is not a valid option for SelectMultipleWidget "field name6".
    at com.twosigma.beakerx.easyform.EasyForm.put(EasyForm.java:315)
    at com.twosigma.beakerx.easyform.EasyForm.put(EasyForm.java:46)
    at script15167983090621327029612.run(script15167983090621327029612.groovy:1)
    at com.twosigma.beakerx.groovy.evaluator.GroovyCodeRunner.run(GroovyCodeRunner.java:71)") - at UserContext.<anonymous> (tests/groovy/easyFormTest.js:228:23)


```

**Pull request merged**